### PR TITLE
Go mod

### PR DESCRIPTION
**Pull request by rchiossi**
Use go mod to manage dependencies and to allow make targets to work out of GOPATH

### CODE REVIEW

**Comment by reaganlo** [comment]
Is this ready for review?

**Comment by rchiossi** [comment]
Yes. I'm might tweak some values (like the binary paths), but the core of it shouldn't change.

**Comment by rchiossi** [comment]
Nice catch.

The second one if not a typo though. I will reword it to make it clearer what I meant.

**Comment by rchiossi** [comment]
> What is the significance of vendor/modules.txt file? Is it auto-generated?

Yes. It is a marker that say that this module (mixer) uses vendored dependencies. It is generated by `go mod vendor`

**Comment by jwakre** [comment]
The tablewriter package was added as a dependency in #679 and will need to be included when this PR is rebased.

@reaganlo Was there something blocking this PR?

**Comment by reaganlo** [comment]
@jwakre Yes this has to be added. Nothing is blocking. Was just waiting for the Mixer code to be come stable with all the dnf changes. I think we are good to go now. Will add the new package and merge this one.

**Comment by reaganlo** [comment]
Closing this PR to open another PR #689 with the additional `tablewriter` package.
